<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsFieldConditionalWidget</class>
 <widget class="QWidget" name="QgsFieldConditionalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>336</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Condtional Format Rules</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="pages">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="mNewButton">
         <property name="text">
          <string>New Rule</string>
         </property>
         <property name="icon">
          <iconset resource="../../images/images.qrc">
           <normaloff>:/images/themes/default/symbologyAdd.png</normaloff>:/images/themes/default/symbologyAdd.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Field</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QListView" name="listView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsFieldComboBox" name="mFieldCombo"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Condition</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="mRuleEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="btnBuildExpression">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../images/images.qrc">
             <normaloff>:/images/themes/default/mIconExpression.svg</normaloff>:/images/themes/default/mIconExpression.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="mDefault1">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(208, 255, 187);</string>
           </property>
           <property name="text">
            <string>abc - 123</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="backColor" stdset="0">
            <color>
             <red>154</red>
             <green>216</green>
             <blue>113</blue>
            </color>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mDefaultButtons</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="mDefault2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 167, 151);</string>
           </property>
           <property name="text">
            <string>abc - 123</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="backColor" stdset="0">
            <color>
             <red>255</red>
             <green>130</green>
             <blue>130</blue>
            </color>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mDefaultButtons</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="mDefault3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(240, 204, 146);</string>
           </property>
           <property name="text">
            <string>abc - 123</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="backColor" stdset="0">
            <color>
             <red>241</red>
             <green>186</green>
             <blue>118</blue>
            </color>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mDefaultButtons</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_7">
           <property name="styleSheet">
            <string notr="true">color: rgb(170, 64, 75);</string>
           </property>
           <property name="text">
            <string>abc - 123</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="fontColor" stdset="0">
            <color>
             <red>188</red>
             <green>54</green>
             <blue>49</blue>
            </color>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mDefaultButtons</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_8">
           <property name="styleSheet">
            <string notr="true">color: rgb(209, 166, 57);</string>
           </property>
           <property name="text">
            <string>abc - 123</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="fontColor" stdset="0">
            <color>
             <red>207</red>
             <green>141</green>
             <blue>61</blue>
            </color>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mDefaultButtons</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_3">
           <property name="styleSheet">
            <string notr="true">color: rgb(98, 136, 78);</string>
           </property>
           <property name="text">
            <string>abc - 123</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="fontColor" stdset="0">
            <color>
             <red>80</red>
             <green>157</green>
             <blue>52</blue>
            </color>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">mDefaultButtons</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <widget class="QGroupBox" name="horizontalGroupBox_2">
         <property name="title">
          <string>Custom</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Background</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsColorButtonV2" name="btnBackgroundColor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Text</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsColorButtonV2" name="btnTextColor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkIcon">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Icon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnChangeIcon">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QToolButton" name="mFontBoldBtn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="toolTip">
               <string>Bold text
(data defined only, overrides Style)</string>
              </property>
              <property name="text">
               <string>B</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mFontButtons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="mFontItalicBtn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <italic>true</italic>
               </font>
              </property>
              <property name="toolTip">
               <string>Italic text
(data defined only, overrides Style)</string>
              </property>
              <property name="text">
               <string>I</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mFontButtons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="mFontUnderlineBtn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <underline>true</underline>
               </font>
              </property>
              <property name="toolTip">
               <string>Underlined text</string>
              </property>
              <property name="text">
               <string>U</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mFontButtons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="mFontStrikethroughBtn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <strikeout>true</strikeout>
               </font>
              </property>
              <property name="toolTip">
               <string>Strikeout text</string>
              </property>
              <property name="text">
               <string>S</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">mFontButtons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="mFontFamilyCmbBx">
              <property name="editable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="mSaveRule">
           <property name="text">
            <string>Done</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mCancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="mDeleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../../images/images.qrc">
             <normaloff>:/images/themes/default/mActionDeleteSelected.svg</normaloff>:/images/themes/default/mActionDeleteSelected.svg</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgscolorbuttonv2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkIcon</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnChangeIcon</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="mFontButtons">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="mDefaultButtons"/>
 </buttongroups>
</ui>
